### ISSUE TITLE
Tabs do not have an initial focus indicator (when you first keyboard tab onto the widget)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/ugnext#212]
DARC feedback:
- "When the focus indicator goes on the 'Tab One Title' there is no focus indicator showing I've now highlighted the first tab, which is confusing"
- "Tabs do not have an initial focus indicator"
<img width="1453" height="767" alt="Image" src="https://github.com/user-attachments/assets/3ab3a035-28bc-45b0-b170-47187cdb1127" />
